Source/Core/DolphinQt/Utils/Resources.cpp

```diff
@@ -78,6 +78,8 @@ void Resources::UpdatePixmaps()
 	// TODO: toolbar[MEMCARD];
 	// TODO: toolbar[HOTKEYS];
 	m_pixmaps[BANNER_MISSING].load(GIFN("nobanner"));
+	// TODO: Make this consistent with the other files
```

Comment:
All the platform files should be in one place. I'm not going to do anything about it in this PR - it involves changing a bunch of wx stuff.